.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!-- Just like NASA going to the moon, it's always good to have a checklist when
+creating changes.
+The following items are listed to help you create a great Pull Request: -->
+## Checklist
+- [ ] All new and existing **tests are passing**
+- [ ] (If adding features:) I have added tests to cover my changes
+- [ ] (If docs changes needed:) I have updated the **documentation** accordingly.
+- [ ] I have added an entry to `CHANGES.rst` because this is a user-facing change or an important bugfix
+- [ ] I have written **proper commit message(s)**
```

Comment:
This is the wrong place for this, it belongs in a CONTRIBUTING.md or something similar; by the time the contributor creates the pull request they've already written these commit messages so this is the wrong moment to find out about the right format.

Reply:
All perfectly valid points. Since docs on posativ.org are outdated for now, maybe we could link to the source doc as rendered here: [isso@master:docs/contributing/index.rst](https://github.com/posativ/isso/blob/master/docs/docs/contributing/index.rst#code-contributions)